### PR TITLE
BandColors, and example implimentation with youtube

### PR DESCRIPTION
**Pull request by zwrawr**
Addresses #144 

Since @Noojuno has made it way easier to mess with the auto-embed code, were probably going to start adding more embeds. Were going to want to add logo's of these sites and style them with the appropriate branding colours. As mentioned in #144 we really don't want to have to go out and find all brand colours ourselves. This is where https://brandcolors.net/ comes in. Unfortunately were dealing with a lot of gaming and development specific sites which aren't included. In that case we have to add the colours our-self.

This commit adds.
* https://brandcolors.net/ style sheet `src/external/BrandColors/brandcolors.less`
* A readme with details with brandcolor site and issues page. *(note: there is no licence on the .less file, probably because all the colors are 'owned' by the brands them selves)*
* Internal extra site colors (itch. gamejolt, 1h game jam) `src/internal/BrandColors/brandcolors.less`
* Makes Youtube logo use the youtube color image

*Ignore the diff for `src/com/autoembed/youtube.less` it's messed up.*

### CODE REVIEW

File: src/com/autoembed/youtube.less [hunk 1]
```diff
@@ -1,101 +1,39 @@
...
+                & > a,
+                & > a:visited {
+                color: @COL_L;
+                }
+            }
+        }
     }
-  }
-}
+}
```
Comment (povrazor) [inline src/com/autoembed/youtube.less]: This should have been caught by the linter. Newline at end of file.

File: src/custom/brandcolors/brandcolors_custom.less [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+@bc-gamejolt-2: #2f7f6f;
+@bc-gamejolt-3: #ff3fac;
+@bc-gamejolt-4: #31d6ff;
+
+// https://hitbox.desk.com/customer/en/portal/articles/1434405-can-i-use-the-hitbox-logo-on-my-stream-overlay-website-etc-?b_id=16071
+@bc-hitbox-tv-1: #9C0;
+
+// Pulled from there logo
+@bc-one-hour-game-jam-1: #ff0066;
+@bc-one-hour-game-jam-2: #4A3457;
```
Comment (povrazor) [inline src/custom/brandcolors/brandcolors_custom.less]: This also should have been caught by the linter. Newline at end of file please.

File: src/com/autoembed/youtube.less [hunk 3]
```diff
@@ -1,101 +1,39 @@
...
+                color: @bc-youtube;
+                background: fade(@COL_D, 50%);
 
-  & > .-video {
-    position: relative;
-    width: 100%;
-    height: 0;
-    padding-bottom: 100% * (9/16);
+                &:hover {
+                background: @bc-youtube;
```
Comment (povrazor) [inline src/com/autoembed/youtube.less]: Indentation. :wink:

(I think there's another one too just a few lines up, but I can't see it in this view)

File: src/com/autoembed/youtube.less [hunk 4]
```diff
@@ -1,101 +1,39 @@
...
 
-    & > iframe {
-      position: absolute;
-      width: 100%;
-      height: 100%;
-      left: 0;
-      top: 0;
+                & > a,
+                & > a:visited {
+                color: @COL_L;
```
Comment (povrazor) [inline src/com/autoembed/youtube.less]: Indentation. :wink:

**Comment by povrazor** [comment]
What would help in the future if 4-wide-tabs were used instead of 2-spaces.

File: src/com/autoembed/youtube.less (hunk 3 above)
Comment (zwrawr) [reply src/com/autoembed/youtube.less]: fixed

File: src/com/autoembed/youtube.less (hunk 4 above)
Comment (zwrawr) [reply src/com/autoembed/youtube.less]: fixed

File: src/com/autoembed/youtube.less (hunk 1 above)
Comment (zwrawr) [reply src/com/autoembed/youtube.less]: fixed

File: src/custom/brandcolors/brandcolors_custom.less (hunk 2 above)
Comment (zwrawr) [reply src/custom/brandcolors/brandcolors_custom.less]: fixed

**Pull request merged**